Updating variant of #F0F0F0

HerrHulaHoop opened the pull request:
When using this dark theme with the wiki of RPCS3 (PS3 emulator) [https://wiki.rpcs3.net/] the infobox escapes the theme due to absence of this #f0f0f0 variant. Hope this is okay.

Here's a [link](https://wiki.rpcs3.net/index.php?title=God_of_War:_Collection) to see this happening.

<details>
 <summary>Before</summary>

![before](https://user-images.githubusercontent.com/31934788/45895638-5a065d00-bdef-11e8-8efe-f506c709c65d.png)
</details>
<details>
 <summary>After</summary>

![after](https://user-images.githubusercontent.com/31934788/45895637-596dc680-bdef-11e8-9511-7673b4128ef6.png)
</details>

Do let me know if anything else is required!

Mottie commented:
Hi @HerrHulaHoop!

Thanks for submitting!

* You can add `domain("wiki.rpcs3.net")` to the `@-moz-document` if you wish to include that site.
* The css selector is unnecessarily over-specific, please change it to just `tr[style*="background:#f0f0f0;" i]` (remove the `table > tbody >`).

HerrHulaHoop commented:
Thanks for the review!

I have addressed your comment and also added wiki.rpcs3.net to the `@-moz-document` section.

Let me know if any further changes are required.

Mottie commented on wikipedia-dark.user.css:
```diff
@@ -163,6 +163,7 @@
   tr[style*="background-color: #f5faff;" i],
   tr[style*="background:#E9E9E9" i] > *,
   table.infobox.hproduct > tbody > tr[style*="background: #F0F0F0;" i],
```
Because wikipedia can be edited by anyone, there is no consistency in styling... I don't remember the exact reason for that selector choice, but I'm guessing there was a reason. But go ahead and remove  the `> *` and we'll keep an eye out for any missing styles.

HerrHulaHoop replied:
Sure I'll remove that but I was more specifically referring to `table.infobox.hproduct > tbody >` in line 165. Is that part necessary?

Mottie replied:
Leave it for now, we can fix it later... It looks it was originally added in #7, altered in https://github.com/StylishThemes/Wikipedia-Dark/commit/c365792cb49d843a5f75f698b25f09d219549217; but I think wikipedia altered the box again, because even removing that line doesn't change the appearance. What do you think about removing the `table.infobox.hproduct > tbody >` @Poorchop?

Poorchop replied:
Sorry for the delayed response, I'm up to my eyes in work. I remember that commit but as you already mentioned, Wikipedia can be edited by anyone and there's no consistency so who even knows if that rule is applicable anymore. I think it's best to spend some time with the modified style or maybe push it to a dev branch until we're reasonably sure that nothing breaks from deleting a rule.

HerrHulaHoop replied:
Alrighty! I managed to pull out the old style from the [infobox history](https://en.wikipedia.org/w/index.php?title=Template:Infobox_video_game&oldid=674824106). The incorrect colours are still perfectly fixed without `table.infobox.hproduct > tbody >`. So I think it should be okay to drop that part. Let me know your thought!

HerrHulaHoop commented:
Alright, for the time being I have removed `> *` as per the above discussion. Based on @Poorchop response, we can deal with the other line. 

Meanwhile, I have found 2 other issues that I have addressed in the new commit:
1. The namespace selection table [here](https://en.wikipedia.org/w/index.php?title=Special:Search&profile=advanced&search=how%20many%20different%20colors%20can%20there%20be%20in%20this%20site&fulltext=1) doesn't respect the dark theme. I've added `#mw-searchoptions` to the appropriate colour to fix this.

<details>
 <summary>Before</summary>

![before1](https://user-images.githubusercontent.com/31934788/46047106-446f9b00-c141-11e8-8028-bc723dd90c4f.png)
</details>
<details>
 <summary>After</summary>

![after1](https://user-images.githubusercontent.com/31934788/46047122-52252080-c141-11e8-9ff0-f27d2bbcd6e4.png)
</details>

2. This table [here](https://en.wikipedia.org/wiki/List_of_horror_films_of_the_1920s) doesn't conform to the dark theme. So I added `th[style*="background:#E9E9E9" i]` in as well to handle it.

<details>
 <summary>Before</summary>

![before2](https://user-images.githubusercontent.com/31934788/46047132-5b15f200-c141-11e8-87aa-9b73a3fb38cb.png)
</details>
<details>
 <summary>After</summary>

![after2](https://user-images.githubusercontent.com/31934788/46047137-61a46980-c141-11e8-94bb-1835887ef1bf.png)
</details>

Mottie commented:
Thanks again!
